src/components/Inputs/EmailInput/EmailInput.js

```diff
@@ -29,6 +29,7 @@ export const EmailInput = (props) => {
         data-tid={restProps['data-tid']}
         placeholder={placeholder}
         validator={EmailFormatValidator}
+        {...restProps}
```

Comment:
I'm passing these through so that the private field stuff gets passed in `currentValue`, `currentError` etc. I'm not sure if this is best idea as those would ideally stay while listed as props..but this gets it working